src/notifications.c

```diff
@@ -355,6 +389,11 @@ void Initialize_KeyspaceNotifications(RedisModuleCtx *ctx) {
     RedisModule_Log(ctx, "notice", "%s", "Subscribe to clear resources on shutdown");
     RedisModule_SubscribeToServerEvent(ctx, RedisModuleEvent_Shutdown, ShutdownEvent);
   }
+
+  if (RedisModule_SubscribeToServerEvent) {

```

Comment:
What if this fails?
Let's add a log, or assertion (I guess it's a compatibility question)

Reply:
Removed the condition since `RedisModule_SubscribeToServerEvent` should exist since redis 6.0